theia-callhierarchy/src/browser/callhierarchy-tree/callhierarchy-tree-widget.ts

```diff
@@ -38,6 +43,20 @@ export class CallHierarchyTreeWidget extends TreeWidget {
         this.title.iconClass = 'fa fa-arrow-circle-down';
         this.title.closable = true;
         this.addClass(HIERARCHY_TREE_CLASS);
+        this.initializeModel();
+    }
+
+    protected initializeModel(): void {
+        const selection = this.editorAccess.getSelection();
+        const languageId = this.editorAccess.getLanguageId();
+        this.model.initializeCallHierarchy(languageId, selection);
```

Comment:
is the `initializeCallHierarchy` call necessary? it's already called in https://github.com/theia-ide/theia-callhierarchy-extension/blob/master/theia-callhierarchy/src/browser/callhierarchy-contribution.ts#L56.